### DISCUSSION TITLE
I am using this package in flutter mobile application

### DISCUSSION THREAD

**Discussion by Muneeb-Ahmed2002** [saran-surya/email_auth#88] [Q&A]
I am not able to create a remote server and I want to add Firebase remote server using Firebase remote Config if there is any possibility of doing so then please let me know I am waiting for it.

**Comment by saran-surya** [comment]
Hi @Muneeb-Ahmed2002 ,
Can you please elaborate your query ?

**Reply by Muneeb-Ahmed2002** [reply]
I am working on my final year project and I am new to flutter I want to design a signup where I have stored my values on firebase.
for email verification firebase provides an email link but with this package we can verify emails via OTP.
I am not having any linkage with github for my project and thus I want some process to create a remote server via firebase 
Firebase also provides remote server config but I am not able to identify the variables that would be used for firebase remote config for configuring my server for the sake of using this project 
I want to ask if there is any way of using firebase remote config with this package to send the OTP  message.

**Reply by saran-surya** [reply]
Okay based on what I understand is that, you want to use this as a plugin with firebase severs ?
This is actually a valid point though, Thing is Google uses the new method of URL verification through emails, as that is the latest method.
As of now the project doesn't support the integration with firebase, I will have a check and let you know regarding it. Maybe if you want to you can use the firebase SDK with a NODE JS application and use them instead. I would still like a clear cut answer, but will surely try to look up and let you know regarding it.

**Comment by Muneeb-Ahmed2002** [comment]
Thank you so much it would be a great thing to know if it happens as of now I am using EmailJS for working but it allows sending one email

**Comment by saran-surya** [comment]
Yes, Actually it is a bit funny to see people use open source to build
tools that are so mentioned "PREMIUM"
I will surely look for a solution and let you know, But it might take some
time. If it is so urgent please let me know we can connect through a G-MEET
and figure out a solution.

Thankyou

**Comment by Muneeb-Ahmed2002** [comment]
Actually we are working on our final-year project and for this we are using firebase already but for signup we don't want email links for setting password and so that is why I was looking for something meanwhile I have found a way to send custorm emails via EmailJS

**Comment by Muneeb-Ahmed2002** [comment]
it is not that urgent I can take a while to wait for it as 2 months are left I am working on other important and functional features for the application.
